### PR TITLE
Tiffany Wu Biography

### PR DESCRIPTION
**Pull request by tiffany-wu**
**Reflection of Process:**
First, I forked the hw01 repository into mine and cloned the url into RStudio. Then, I started writing a rough draft of my biography. I initially had trouble finding where to switch from R Script to Markdown in the script box on R studio, so I first tried the different formatting syntax using Dingus (from the Daring Fireball website). The syntax cheatsheet also helped me remember how to format everything when I figured out how to switch to Markdown on the bottom right-hand side, but overall the syntax was pretty intuitive. 

I ran into 2 main problems. The first problem was embedding my image after uploading it into myrepo. I kept getting an error message and the image wouldn’t show up. A huge thank you to Mayank who helped me realize I was copying the wrong url from my repo in GitHub. My second problem was figuring out which files to commit and push. I had the project file, a .gitignore file, and two .md files (I accidentally made a new file before realizing I wasn’t editing the original readme file), and I didn’t know which ones were relevant. Again, thanks to Mayank I realized I had to just upload the readme.md file.

**URL to HW01**
https://github.com/tiffany-wu/hw01.git 

**Helpful Links:**
https://daringfireball.net/projects/markdown/dingus
https://guides.github.com/features/mastering-markdown/#examples

### CODE REVIEW

**Comment by jiaqianwang18** [comment]
Evaluation HW-01 of Tiffany Wu
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     √     |              |            |
| **Coding strategy**       |     √     |              |            |
| **Presentation**          |     √      |              |            |
| **Achievement, creativity** |   √      |              |            |
| **Ease of access**          |    √     |              |            |

Remarks:

* Overall, this is **Terrific**!
* Code: Code is efficient, correct, and minimal. Code runs! Besides, Tiffany demonstrated her proficiency with headers, emphasis, lists, links and images. For one thing, the code she wrote was accurate. For another, from the image she inserted to the links she posted, the styling she employed enhanced the presentation. Excellent!
* Content: The biography provides a very comprehensive and vivid background of Tiffany. I can infer that she treated this assignment as not only an assignment but also an opportunity to let us know her. The rich content reflects her extraordinary efforts. Impressive!
* Reflection: The reflection clearly introduced the process and problems in finishing the assignment. Also, Tiffany provided some helpful links which I would take a look at. Thanks!

**Comment by leonawu0723** [comment]
### Evaluation of HW1 for Tiffany Wu
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x      |              |            |
| **Coding strategy**         |    x      |              |            |
| **Presentation**            |    x      |              |            |
| **Achievement, creativity** |    x      |              |            |
| **Ease of access**          |    x      |              |            |

Remarks:
* Tiffany did a fantastic job on this assignment. Her codes were well organized and included many variations. The work is highly presentable and visually satisfying. She organized the content of her biography very coherently and logically. She decided to put links where readers would get curious. For example, she put a link of her research work in a psychology laboratory, which helps the readers learn more about her past experiences and research interests. And the links all work perfectly. She also put different elements of the Markdown language where it fits. For example, the book titles were in italics. The code was reproducible as well.  
* She chose her picture very wisely. It is of an appropriate size and quality, and well illustrates her background. I also really enjoyed the link for interspecies animal friendship, that was very cute and creative. 
* What I learned from Tiffany's work are strategies to organize the flow of my biography. Specifically, how to choose appropriate headings, insert useful links, and be creative.   
* I think her work met all the points with added creativity, and thus I do not have any specific criticism.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
-# hw01
-Repository for homework 01
+Tiffany Wu's Biography
+=======================
```
Comment (bensoltoff) [inline README.md:2]: Alternatively you can use `#` symbols to denote headers

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+Background Information
+----------------------
+Although my parents are both Taiwanese, I was born in the Chicago suburbs and have lived in Illinois all my life. I did my undergrad at Northwestern University, which has a bit of a (possibly imaginary) rivalry with UChicago. I was a history and secondary education major and Spanish minor, focusing on ways to improve urban education. During this time I also worked as a RA in Dr. Lindsay Chase-Lansdale's lab on the [Two-Generation Research Initiative for Head Start in Tulsa, Oklahoma](https://www.ipr.northwestern.edu/about/news/2017/chase-lansdale-two-generation.html). 
+
+![](https://github.com/tiffany-wu/myrepo/blob/master/TIffany_Northwestern.png?raw=true)
```
Comment (bensoltoff) [inline README.md:7]: Unless there is a specific need to store in a separate repo, probably a better idea to put it in here as well. Then your code would be ``

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Nice clean repo - only the required `README.md` file uploaded
* Lots of different Markdown elements

**Pull request closed without merging**